frappe/core/doctype/system_settings/system_settings.json

```diff
@@ -452,12 +453,19 @@
    "fieldname": "prepared_report_section",
    "fieldtype": "Section Break",
    "label": "Prepared Report"
+  },
+  {
+   "default": "$ 10",
+   "fieldname": "currency_indicator_style",
+   "fieldtype": "Select",
+   "label": "Currency Indicator Style",
+   "options": "$ 10\n10 USD"
```

Comment:
I think we should cover all four types: $ 10, 10 USD, USD 10, 10 $

The actual value should be a named pattern. For e.g., "Value Currency"

So we can have four values 
- "Currency Value" (USD 10)
- "Value Currency" (10 USD)
- "Symbol Value" ($ 10)
- "Value Symbol" (10 $)

We can show the rendered version as the label of the select option in UI.

For e.g., The select option would be "USD 10" but the field value should be set as "Currency Value"